setup.py

```diff
@@ -27,8 +27,8 @@
 
 
 def get_version():
-    version_file = CURDIR / "src" / "pipx" / "version.py"
-    namespace = run_path(version_file)
+    version_file = Path(CURDIR, "src", "pipx", "version.py")
```

Comment:
`CURDIR` is a path,  so there is no need to recast it. `version_file` was already a Path in the original code.